src/index.html

```diff
@@ -19,0 +1,15 @@
+<!DOCTYPE html>
+<div class="product-card" data-qa="card">
+  <div class="product-card__image product-card__image--product1"></div>
+  <div class="product-card__details">
+    <h2 class="product-card__title">Product Title</h2>
+    <div class="product-card__stars">
+      <!-- Stars block here -->
+    </div>
```

Comment:
The `stars` block is mentioned but not implemented. Ensure that the `stars` block is correctly implemented and separated into its own file according to the task requirements.